cmake/UnitTests.cmake

```diff
@@ -150,6 +150,74 @@ list(APPEND FAILING_FOR_ALL "Unit_deviceFunctions_CompileTest_atomicXor_system_u
 list(APPEND FAILING_FOR_ALL "hipStreamSemantics") # SEGFAULT - likely due to main thread exiting without calling join
 list(APPEND FAILING_FOR_ALL "Unit_hipMultiStream_multimeDevice") # SEGFAULT - likely due to multiple GPU support
 
+list(APPEND FAILING_FOR_ALL "Unit_hipGraphAddDependencies_NegTest") # (Failed)
+list(APPEND FAILING_FOR_ALL "Unit_hipGraphAddEventRecordNode_MultipleRun") # subprocess aborted
+list(APPEND FAILING_FOR_ALL "Unit_hipGraphAddEventWaitNode_MultipleRun") # subprocess aborted
+list(APPEND FAILING_FOR_ALL "Unit_hipGraph_BasicFunctional") # subprocess aborted
+list(APPEND FAILING_FOR_ALL "Unit_hipGraph_SimpleGraphWithKernel") # subprocess aborted
```

Comment:
lots of failing tests for graphs still? Even the most basic tests such as `Unit_hipGraph_BasicFunctional`?

Reply:
yeah about 50% of graph tests still fail, the work is not finished